Flow.Launcher.Core/Plugin/PluginAssemblyLoader.cs

```diff
@@ -33,10 +38,7 @@ protected override Assembly Load(AssemblyName assemblyName)
         {
             string assemblyPath = dependencyResolver.ResolveAssemblyToPath(assemblyName);
 
-            // When resolving dependencies, ignore assembly depenedencies that already exits with Flow.Launcher.Plugin
-            // Otherwise will get unexpected behaviour with plugins, e.g. JsonIgnore attribute not honored in WebSearch or other plugins
-            // that use Newtonsoft.Json
-            if (assemblyPath == null || ExistsInReferencedPluginPackage(assemblyName))
```

Comment:
why remove this comment, we should add an explanation as to why this needs to be done

Reply:
Oh I misunderstood the comment, will bring it back.

Reply:
may be reword the example since we dont use JsonIgnore now

Reply:
Yes